### PR TITLE
remove /etc/systemd/system/uyuni-server.service.d/Service.conf during uninstallation

### PR DESCRIPTION
**Pull request by mbussolotto**
## What does this PR change?

See the title 

## Test coverage

- [ ] **DONE**

## Links

Issue(s): #

- [ ] **DONE**

## Changelogs

Make sure the changelogs entries you are adding are compliant with https://github.com/uyuni-project/uyuni/wiki/Contributing#changelogs and https://github.com/uyuni-project/uyuni/wiki/Contributing#uyuni-projectuyuni-repository

If you don't need a changelog check, please mark this checkbox:

- [ ] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

# Before you merge

Check [How to branch and merge properly](https://github.com/uyuni-project/uyuni/wiki/How-to-branch-and-merge-properly)!

### CODE REVIEW

File: shared/podman/systemd.go [hunk 1]
```diff
@@ -39,16 +39,23 @@ func GetServicePath(name string) string {
 	return path.Join(servicesPath, name+".service")
 }
 
+// GetServiceConfPath return the path for Service.conf file.
+func GetServiceConfPath(name string) string {
+	return path.Join(servicesPath, name+".service.d", "Service.conf")
```
[the remark this answers is not in the document]
Comment (mbussolotto) [reply shared/podman/systemd.go:44]: For sure the folder should be removed if there're no files, I'm going to add this change.
I'm not sure if we should remove the user-added file: I think we should have the same behaviour of command like `zypper uninstall` and I think the user file are not deleted by that command.
What do you think?

**Comment by mbussolotto** [comment]
> I'm fine with this PR, but where do we create that file? Shouldn't we also remove the other config files there too?

`Service.conf` is the only file created by `mgradm` (besides `uyuni-server.service` and `uyuni-server-attestation.service` in coco attestation container) .  `uyuni-server.service.d` might contains other files created by the user...should we remove them during uinstallation?

File: shared/podman/systemd.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (cbosdo) [reply shared/podman/systemd.go:44]: We can leave them, but a new install after would be polluted by those... not sure what's best.

**Comment by mbussolotto** [comment]
@cbosdo Check if this might be a good option:
- folder is removed if it's empty
- a warning message is printed if folder contains user files

**Review by cbosdo — APPROVED** [review]

**Pull request merged**